README.md

```diff
@@ -1,4 +1,6 @@
 
```

Comment:
We should use `README.md` based onto same template across SDKs (the one provided by documentation team)

Reply:
@JPPortier You're right. Maybe let's merge this PR and I'm going to take that task to the next sprint . WDYT?